cpu/kinetis_common/periph/pwm.c

```diff
@@ -35,6 +35,22 @@ static inline FTM_Type *ftm(pwm_t pwm)
     return pwm_config[pwm].ftm;
 }
 
+static void poweron(pwm_t pwm)
```

Comment:
minor and discutable: `poweron` seems to be an internal helper, I would rename it `_poweron` to make this obvious. If accepted, there are other places where this should be changed.